packaging/snap/snapcraft.yaml

```diff
@@ -22,7 +22,7 @@ parts:
     install: |
       # FIXME: downloading with a source: field doesn't work as snapcraft
       # expects the downloaded file to be an archive it can extract.
-      wget https://s3.eu-central-1.amazonaws.com/anbox/images/anbox-android-1.img
+      wget http://build.anbox.io/android-images/01/android_amd64.img
```

Comment:
I would propose to encode date and android version in the URL, e.g.:

android-images/dd/mm/yyyy/${ANDROID_VERSION}/android_amd64.img